test/functional/rpc_utxosnapshot.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
```

Comment:
Let's name the file `rpc_dumptxoutset.py` to match Bitcoin.

Reply:
Do you want to rename the rpc also? This file is named `rpc_utxosnapshot.py` to match the rpc name, `utxosnapshot`. In bitcoin both are `dumptxoutset`.

Reply:
Yes, let's make them both the same.